specification/metrics/pricingquantity.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Quantity
+
+The Pricing Quantity represents the volume of a given resource and/or service used or purchased based on the [Pricing Measure](#pricingmeasure). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not resource and service consumption.
```

Comment:
> a given resource and/or service used or purchased

I wonder whether we can specifically refer to the SKU concept: "a given SKU used or purchased".

Reply:
I believe we purchase resources and/or services, and the underlying SKUs (one or many) are metered and charged.